packages/build/src/plugins_core/build_command.js

```diff
@@ -18,32 +22,39 @@ const coreStep = async function ({
     build: { command: buildCommand, commandOrigin: buildCommandOrigin },
   },
 }) {
-  logBuildCommandStart(logs, buildCommand)
+  return tracer.withActiveSpan('build.command', async (span) => {
+    span.setAttributes({
+      'build.command': buildCommand,
```

Comment:
Could this potentially hold sensitivy information? Or do we sanitize this somehow already?

Reply:
Nope I don't think so. we capture the build command already on multiple places but we are missing it in the traces to build queries from it.

Reply:
> multiple places

Mind elaborating on that? Where, specifically? I would strongly advise against exposing this right now since there's no guarantees to what customers can expose or write in their build commands.